.github/workflows/dependabot-bundler.yaml

```
@@ -0,0 +1,29 @@
+name: Dependabot Bundler
+
+on: workflow_dispatch
+
+jobs:
+  bundler:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

All other workflows are now using v3.
```suggestion
        uses: actions/checkout@v3
```